src/axom/core/tests/CMakeLists.txt

```diff
@@ -60,6 +60,16 @@ blt_add_executable( NAME       core_serial_tests
                     DEPENDS_ON ${utils_tests_depends}
                     FOLDER     axom/core/tests )
 
+# Separate Array tests due to invalid device error with axom::for_all tests
+blt_add_executable( NAME       core_array_test
+                    SOURCES    core_array.cpp
+                    OUTPUT_DIR ${TEST_OUTPUT_DIRECTORY}
+                    DEPENDS_ON ${utils_tests_depends}
+                    FOLDER     axom/core/tests )
+
+axom_add_test( NAME    core_array
+               COMMAND core_array_test )
+
```

Comment:
Thanks for the comment.
Is this still an issue?

If so, please create a ticket so we can address this in the future.